tensorboard/plugins/debugger_v2/tf_debugger_v2_plugin/actions/debugger_actions.ts

```diff
@@ -47,3 +50,42 @@ export const debuggerRunsRequestFailed = createAction(
  * Actions for the Alerts Component.
  */
 export const alertsViewLoaded = createAction('[Debugger] Alerts View Loaded');
+
+/**
+ * Actions for the Timeline Component.
+ */
+export const activeRunIdChanged = createAction(
+  '[Debugger] Debugger Active Run Changed',
+  props<{activeRunId: string | null}>()
+);
+
+export const numExecutionsRequested = createAction(
+  '[Debugger] Number of Executions Requested'
+);
+
+export const numExecutionsLoaded = createAction(
+  '[Debugger] Number of Executions Loaded',
+  props<{numExecutions: number}>()
+);
+
+export const requestExecutionDigests = createAction(
+  '[Debugger] Request for ExecutionDigests',
+  props<{runId: string; begin: number; end: number; pageSize: number}>()
```

Comment:
In effects, you are using `props.pageLoadedSizes` which SHOULD violate the typing here but it is not reporting as such. Can you check why? Also, an action should try hard not to contain any data that is accessible from the store directly (it should have user's inputs)